src/openzaak/components/catalogi/tests/management/test_import_export.py

```diff
@@ -0,0 +1,274 @@
+import json
+import os
+import zipfile
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase
+
+from ...models import (
+    BesluitType,
+    Catalogus,
+    Eigenschap,
+    InformatieObjectType,
+    RolType,
+    StatusType,
+    ZaakInformatieobjectType,
+    ZaakType,
+)
+from ..factories import (
+    BesluitTypeFactory,
+    CatalogusFactory,
+    EigenschapFactory,
+    InformatieObjectTypeFactory,
+    RolTypeFactory,
+    StatusTypeFactory,
+    ZaakInformatieobjectTypeFactory,
+    ZaakTypeFactory,
+)
+
+PATH = os.path.abspath(os.path.dirname(__file__))
+
+
+class ExportCatalogiTests(TestCase):
+    def setUp(self):
+        self.filepath = os.path.join(PATH, "export_test.zip")
+
+    def test_export_catalogus(self):
+        catalogus = CatalogusFactory.create()
+
+        call_command(
+            "export", self.filepath, resource=[["Catalogus"]], ids=[[str(catalogus.id)]]
```

Comment:
the arguments are weird, should be fixed in the command itself I think and then adapt the calling code

```suggestion
            "export", self.filepath, resource=["Catalogus"], ids=[[catalogus.id]]
```

Command should probably also validate that there are equal amounts of elements in `resource` as in `ids`